forc-plugins/forc-client/src/op/deploy.rs

```diff
@@ -179,3 +256,105 @@ fn build_opts_from_cmd(cmd: &cmd::Deploy) -> pkg::BuildOpts {
         member_filter: pkg::MemberFilter::only_contracts(),
     }
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+
+    fn setup_manifest_files() -> BTreeMap<String, PackageManifestFile> {
+        let mut contract_to_manifest = BTreeMap::default();
+
+        let manifests_dir = PathBuf::from(format!(
+            "{}/test/data/manifests",
```

Comment:
Always good to prefer using `join` over `format!` - it's just as easy and supports more platforms, e.g.
```rust
Path::from(env!("CARG_MANIFEST_DIR")).join("test").join("data").join("manifests")
```